.travis.yml

```diff
@@ -3,18 +3,16 @@ cache:
   directories:
     - node_modules
 notifications:
+  slack: dadi:pnhiL60xOrm7GOglHUmb7xHK
   email: false
 node_js:
-  - 'node'
-  - 'lts/*'
   - '8'
+  - '10'
+  - 'lts/*'
 before_script:
   - npm prune
-after_success:
-  - npm run semantic-release
 branches:
   except:
     - /^v\d+\.\d+\.\d+$/
 services:
```

Comment:
Strikes me as odd that we start up MongoDB when testing this... @eduardoboucas I don't think we need this, right?

Reply:
Really odd! I'd say remove it.